--- kdenetwork-3.5.5/ksirc/dsirc.orig	2007-08-19 20:26:45.000000000 +0200
+++ kdenetwork-3.5.5/ksirc/dsirc	2007-08-19 20:26:55.000000000 +0200
@@ -114,8 +114,8 @@
 $nick=$opt_n || $ARGV[0] || $ENV{"SIRCNICK"} || $ENV{"IRCNICK"} || $username;
 $set{"FINGER"}=$ENV{"IRCFINGER"} || "keep your fingers to yourself";
 $set{"USERINFO"}=$ENV{"USERINFO"} || "yep, I'm a user";
-if ($server =~ /^\[([^\]]+)\]:([0-9]*):?([^:]*)$/
-    or $server =~ /^([^:]+):([0-9]*):?([^:]*)$/)
+if ($server =~ /^\[([^\]]+)\]:([0-9]*):?(.*)$/
+    or $server =~ /^([^:]+):([0-9]*):?(.*)$/)
 {
 	($server, $port, $pass)=($1, $2, $3);
 }
