--- kdeaccessibility-3.5.9.orig/kttsd/kcmkttsmgr/kcmkttsd.desktop
+++ kdeaccessibility-3.5.9/kttsd/kcmkttsmgr/kcmkttsd.desktop
@@ -1,5 +1,4 @@
 [Desktop Entry]
-Encoding=UTF-8
 Type=Application
 Name=Text-to-Speech
 Name[bg]=Синтез на глас
@@ -97,7 +96,7 @@
 StartupWMClass=
 Exec=kcmshell kcmkttsd
 Icon=kttsd
-Categories=Qt;KDE;X-KDE-settings-accessibility;
+Categories=Qt;KDE;Utility;Accessibility;
 Keywords=kttsd,kcmkttsmgr,kttsmgr,tts,ttsd,ktts,text,to,speech,speak,synthesizer,synth,festival,command,freetts,proklam,via,voice
 Keywords[bg]=синтез, глас, говор, текст, kttsd, kcmkttsmgr, kttsmgr, tts, ttsd, ktts, text, to, speech, speak, synthesizer, synth, festival, command, freetts, proklam, via, voice
 Keywords[br]=kttsd,kcmkttsmgr,kttsmgr,tts,ttsd,ktts,skrid,da,prezegenn,komz,synthesizer,synth,festival,urzhiad,freetts,proklam,via,mouezh
--- kdeaccessibility-3.5.9.orig/kttsd/kttsmgr/kttsmgr.desktop
+++ kdeaccessibility-3.5.9/kttsd/kttsmgr/kttsmgr.desktop
@@ -1,5 +1,4 @@
 [Desktop Entry]
-Encoding=UTF-8
 Type=Application
 Name=kttsmgr
 Name[pt_BR]=Gerenciador ktts
@@ -93,6 +92,6 @@
 Comment[uk]=Менеджер KDE для синтезу мовлення з тексту
 Comment[vi]=Trình quản lý Văn bản sang Tiếng nói KDE
 Comment[zh_TW]=KDE 文字轉語音管理員
-MimeTypes=
+MimeType=
 Terminal=false
 Categories=Qt;KDE;Utility;Accessibility;
--- kdeaccessibility-3.5.9.orig/ksayit/src/ksayit.desktop
+++ kdeaccessibility-3.5.9/ksayit/src/ksayit.desktop
@@ -1,5 +1,4 @@
 [Desktop Entry]
-Encoding=UTF-8
 Type=Application
 Exec=ksayit -caption "%c" %i %m
 Icon=ksayit
