Index: kdeutils-3.5.9/ark/ark.desktop
===================================================================
--- kdeutils-3.5.9.orig/ark/ark.desktop	2008-07-01 13:03:32.000000000 +0200
+++ kdeutils-3.5.9/ark/ark.desktop	2008-07-01 13:03:37.000000000 +0200
@@ -79,8 +79,6 @@
 DocPath=ark/index.html
 Type=Application
 Terminal=false
-SwallowExec=
-SwallowTitle=
 X-DCOP-ServiceType=Unique
 X-KDE-HasTempFileOption=true
 Categories=Qt;KDE;Utility;X-KDE-Utilities-File;
Index: kdeutils-3.5.9/kdf/kcmdf.desktop
===================================================================
--- kdeutils-3.5.9.orig/kdf/kcmdf.desktop	2008-07-01 13:05:15.000000000 +0200
+++ kdeutils-3.5.9/kdf/kcmdf.desktop	2008-07-01 13:05:28.000000000 +0200
@@ -133,4 +133,4 @@
 Comment[zh_TW]=顯示連接的設備的磁碟使用率
 Comment[zu]=Ikhombisa ukusebenziswa kwediski yamathuluzi okunyuka
 
-Categories=Qt;KDE;X-KDE-information;
+Categories=Qt;KDE;System;X-KDE-information;
Index: kdeutils-3.5.9/kmilo/kmilo_kvaio/kcmkvaio/kvaio.desktop
===================================================================
--- kdeutils-3.5.9.orig/kmilo/kmilo_kvaio/kcmkvaio/kvaio.desktop	2008-07-01 13:01:58.000000000 +0200
+++ kdeutils-3.5.9/kmilo/kmilo_kvaio/kcmkvaio/kvaio.desktop	2008-07-01 13:02:11.000000000 +0200
@@ -143,4 +143,4 @@
 Keywords[uk]=Sony, Vaio, kvaio, яскравість, вимкнення звуку
 Keywords[zh_CN]=Sony, Vaio, kvaio, brightness, mute, Jog Dial,索尼,亮度,静音
 
-Categories=Qt;KDE;X-KDE-settings-system;
+Categories=Qt;KDE;Settings;X-KDE-settings-system;
Index: kdeutils-3.5.9/kmilo/thinkpad/kmilo_thinkpad.desktop
===================================================================
--- kdeutils-3.5.9.orig/kmilo/thinkpad/kmilo_thinkpad.desktop	2008-07-01 13:02:33.000000000 +0200
+++ kdeutils-3.5.9/kmilo/thinkpad/kmilo_thinkpad.desktop	2008-07-01 13:03:09.000000000 +0200
@@ -97,3 +97,4 @@
 Comment[zh_CN]=此插件启用各种键盘和便携电脑的特殊按键
 Comment[zh_TW]=開啟特殊 Thinkpad 按鍵支援
 X-KDE-Library=kmilo_thinkpad
+Categories=Qt;KDE;Settings;X-KDE-settings-system;
Index: kdeutils-3.5.9/kwallet/konfigurator/kwalletconfig.desktop
===================================================================
--- kdeutils-3.5.9.orig/kwallet/konfigurator/kwalletconfig.desktop	2008-07-01 13:00:59.000000000 +0200
+++ kdeutils-3.5.9/kwallet/konfigurator/kwalletconfig.desktop	2008-07-01 13:01:11.000000000 +0200
@@ -146,4 +146,4 @@
 Keywords[uk]=Торбинка,Заповнення форм,Паролі,Дані форм
 Keywords[zh_CN]=Wallet,Form Fill,Passwords,Form Data,钱包,填充表单,密码,表单数据
 
-Categories=Qt;KDE;X-KDE-settings-security;
+Categories=Qt;KDE;Settings;X-KDE-settings-security;
