Index: kdeutils/ark/rar.cpp
===================================================================
--- kdeutils/ark/rar.cpp
+++ kdeutils/ark/rar.cpp
@@ -55,6 +55,8 @@
 {
   // Check if rar is available
   bool have_rar = !KGlobal::dirs()->findExe( "rar" ).isNull();
+  bool have_unrar = !KGlobal::dirs()->findExe( "unrar" ).isNull();
+  bool have_unrar_free = !KGlobal::dirs()->findExe( "unrar-free" ).isNull();
 
   if ( have_rar )
   {
@@ -63,13 +65,20 @@
     verifyCompressUtilityIsAvailable( m_archiver_program );
     verifyUncompressUtilityIsAvailable( m_unarchiver_program );
   }
-  else
+  else if (have_unrar)
   {
     // If rar is not available, try to use unrar to open the archive read-only
     m_unarchiver_program = "unrar";
     verifyUncompressUtilityIsAvailable( m_unarchiver_program );
     setReadOnly( true );
   }
+  else
+  {
+    // If rar is not available, try to use unrar to open the archive read-only
+    m_unarchiver_program = "unrar-free";
+    verifyUncompressUtilityIsAvailable( m_unarchiver_program );
+    setReadOnly( true );
+  }
 
 
 
