diff -Nurp k3b-1.0rc4-clean/src/k3bapplication.cpp k3b-1.0rc4/src/k3bapplication.cpp
--- k3b-1.0rc4-clean/src/k3bapplication.cpp	2007-01-09 13:36:40.000000000 +0000
+++ k3b-1.0rc4/src/k3bapplication.cpp	2007-01-09 16:39:51.000000000 +0000
@@ -143,7 +143,7 @@ void K3bApplication::init()
 
     emit initializationDone();
 
-    K3bFirstRun::run( m_mainWindow );
+    //K3bFirstRun::run( m_mainWindow );
 
     if( K3bSystemProblemDialog::readCheckSystemConfig() ) {
       emit initializationInfo( i18n("Checking System") );
