diff -Nurp k3b-1.0rc5-clean/src/k3bsystemproblemdialog.cpp k3b-1.0rc5/src/k3bsystemproblemdialog.cpp
--- k3b-1.0rc5-clean/src/k3bsystemproblemdialog.cpp	2007-01-27 11:27:49.000000000 +0000
+++ k3b-1.0rc5/src/k3bsystemproblemdialog.cpp	2007-02-05 17:00:52.000000000 +0000
@@ -580,7 +580,7 @@ void K3bSystemProblemDialog::checkSystem
   }
   if( problems.isEmpty() ) {
     kdDebug() << "          - none - " << endl;
-    K3bPassivePopup::showPopup( i18n("No problems found in system configuration."), i18n("System Problems") );
+    //K3bPassivePopup::showPopup( i18n("No problems found in system configuration."), i18n("System Problems") );
   }
   else {
     static K3bSystemProblemDialog* s_openDlg = 0;
