diff -Nur k3b-1.0.2/src/k3bsystemproblemdialog.cpp k3b-1.0.2.new/src/k3bsystemproblemdialog.cpp
--- k3b-1.0.2/src/k3bsystemproblemdialog.cpp	2007-07-12 11:47:03.000000000 +0200
+++ k3b-1.0.2.new/src/k3bsystemproblemdialog.cpp	2007-07-12 11:48:00.000000000 +0200
@@ -518,8 +518,8 @@
 					      "include Mp3 support for legal reasons."),
 					 i18n("To enable Mp3 support, please install the MAD Mp3 decoding library as well as the "
 					      "K3b MAD Mp3 decoder plugin (the latter may already be installed but not functional "
-					      "due to the missing libmad). Some distributions allow installation of Mp3 support "
-					      "via an online update tool (i.e. SuSE's YOU)."),
+					      "due to the missing libmad). To get Mp3 support, please install libk3b3-extracodecs"
+					      "(or kubuntu-restricted-extras on Kubuntu)."),
 					 false ) );
   }
 
