--- a/kdeprint/cups/cupsdconf2/cupsddialog.cpp
+++ b/kdeprint/cups/cupsdconf2/cupsddialog.cpp
@@ -206,14 +206,9 @@
 	else
 	{
                 bool success = false;
-                if (getuid() == 0 )
-                        success = (::kill(serverPid, SIGHUP) == 0);
-                else
-                {
-                        KProcess proc;
-                        proc << "kdesu" << "-c" << "kill -SIGHUP " + QString::number( serverPid );
-                        success = proc.start( KProcess::Block ) && proc.normalExit();
-                }
+		KProcess proc;
+		proc << "kdesu" << "-c" << "/etc/init.d/cups restart";
+		success = proc.start( KProcess::Block ) && proc.normalExit();
                 if( !success )    
 			msg = i18n("Unable to restart CUPS server (pid = %1)").arg(serverPid);
 	}
