--- kdelibs/kdecore/kdebug.cpp
+++ kdelibs.new/kdecore/kdebug.cpp
@@ -238,7 +238,8 @@
       break;
   }
 
-  short nOutput = kDebug_data->config ? kDebug_data->config->readNumEntry(key, 2) : 2;
+  // if no output mode is specified default to no debug output
+  short nOutput = kDebug_data->config ? kDebug_data->config->readNumEntry(key, 4) : 4;
 
   // If the application doesn't have a QApplication object it can't use
   // a messagebox.
--- kdelibs/kdecore/kdebugrc
+++ kdelibs.new/kdecore/kdebugrc
@@ -86,7 +86,7 @@
 
 # KMail
 [5006]
-InfoOutput=2
+InfoOutput=4
 
 # KitchenSync (Syncing Algorithm)
 [5250]
