Index: kdelibs/kioslave/http/http.cc
===================================================================
--- kdelibs/kioslave/http/http.cc	(revisión: 860100)
+++ kdelibs/kioslave/http/http.cc	(revisión: 860101)
@@ -2753,7 +2753,7 @@
     // calling application later
     m_responseHeaders << QString::fromLatin1(buf);
 
-    if ((strncasecmp(buf, "HTTP", 4) == 0) ||
+    if ((strncasecmp(buf, "HTTP/", 5) == 0) ||
         (strncasecmp(buf, "ICY ", 4) == 0)) // Shoutcast support
     {
       if (strncasecmp(buf, "ICY ", 4) == 0)
