--- a/kdm/kfrontend/kgreeter.cpp
+++ b/kdm/kfrontend/kgreeter.cpp
@@ -364,7 +364,6 @@
 KGreeter::putSession( const QString &type, const QString &name, bool hid, const char *exe )
 {
 	int prio = exe ? (!strcmp( exe, "default" ) ? 0 :
-	                  !strcmp( exe, "custom" ) ? 1 :
 	                  !strcmp( exe, "failsafe" ) ? 3 : 2) : 2;
 	for (uint i = 0; i < sessionTypes.size(); i++)
 		if (sessionTypes[i].type == type) {
@@ -392,7 +391,6 @@
 			}
 	}
 	putSession( "default", i18n("Default"), false, "default" );
-	putSession( "custom", i18n("Custom"), false, "custom" );
 	putSession( "failsafe", i18n("Failsafe"), false, "failsafe" );
 	qBubbleSort( sessionTypes );
 	for (uint i = 0; i < sessionTypes.size() && !sessionTypes[i].hid; i++) {
