--- a/konversation/src/config/preferences.cpp	2007-07-17 20:45:01.000000000 +0300
+++ b/konversation/src/config/preferences.cpp	2007-07-17 21:06:23.000000000 +0300
@@ -68,14 +68,14 @@
     mIdentity->setNicknameList(nickList);
 
     Konversation::ServerGroupSettingsPtr serverGroup = new Konversation::ServerGroupSettings;
-    serverGroup->setName("Freenode");
+    serverGroup->setName("Debian IRC");
     Konversation::ServerSettings server;
-    server.setHost("irc.freenode.org");
-    server.setPort(8001);
+    server.setHost("irc.debian.org");
+    server.setPort(7000);
     serverGroup->addServer(server);
     serverGroup->setIdentityId(mIdentity->id());
     Konversation::ChannelSettings channel;
-    channel.setName("#kde");
+    channel.setName("#debian");
     serverGroup->addChannel(channel);
     serverGroup->setExpanded(false);
     addServerGroup(serverGroup);
